docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,68 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
```

Comment:
Should this be like a bulleted list or something? It almost reads like one.

Reply:
Do you mean something like this?

> The current release version of the ASO deployment YAML installs the operator in single-tenant mode. It:
> * is deployed in the `azureserviceoperator-system` namespace,
> * has a single set of Azure credentials,
> * manages Azure resources in any namespace,
> * handles webhooks fired when any Azure resource is changed.

That feels a bit clunky to me, but if you think it's clearer I'll update it.

Reply:
I did mean that, but agree it is a bit clunky so maybe just leave it as is.